src/common-signals.js

```diff
@@ -27,24 +27,35 @@ export const CommonSignals = {
   BUILT: 'built',
 
   /**
-   * The initial contents of an element/document/embed have been loaded.
+   * The element has started loading.
+   * LOAD_START triggers at the start of the layoutCallback.
    */
-  INI_LOAD: 'ini-load',
+  LOAD_START: 'load-start',
 
   /**
-   * The element has been loaded.
+   * Rendering has been confirmed to have been started.
+   * RENDER_START is an optional signal, implemented by ads, shadowdoc.
+   * The signal instruct the content has started rendering, and is related
+   * to toggle visibility.
    */
-  LOAD_END: 'load-end',
+  RENDER_START: 'render-start',
 
   /**
-   * The element has started loading.
+   * The element has been loaded.
+   * LOAD_END triggers at the end of the layoutCallback.
+   *
    */
-  LOAD_START: 'load-start',
+  LOAD_END: 'load-end',
 
   /**
-   * Rendering has been confirmed to have been started.
+   * The initial contents of an element/document/embed have been loaded.
+   * INI_LOAD is an optional signal, implemented by ads, story.
+   * It instructs that all critical resources has been loade, and is related
```

Comment:
give an example what feature is listening to this singal

Reply:
I put "and can be used for more accurate visibility measurement."